src/components/rating/rating.fluent.scss

```diff
@@ -1,9 +1,12 @@
 @use '../../styles/utilities' as *;
 
-$disabled-color: color(gray, 200) !default;
-$disabled-selected-color: color(gray, 500) !default;
 $disabled-label-color: color(gray, 400) !default;
 
+:host {
+    --symbol-full-color: #{color(warn, 900)};
+    --symbol-empty-color: #{color(gray, 700)};
```

Comment:
Shouldn't we reflect this in the schema for the rating component as well?

Reply:
Yes, but we can do that when we consume the schema colors for the raring.